src/specs/data_structures.md

```diff
@@ -464,26 +463,16 @@ Finally, the `availableDataRoot` of the block [Header](#header) is computed as t
 
 ### Invalid Erasure Coding
 
-If a malicious block producer incorrectly computes the 2D Reed-Solomon code for a block's data, a fraud proof for this can be presented.
+If a malicious block producer incorrectly computes the 2D Reed-Solomon code for a block's data, a fraud proof for this can be presented. Data type for the fraud proof can be found in [here](./networking.md#badencodingfraudproof).
```

Comment:
Actually, this entire section on invalid erasure coding can be moved to networking.md

Reply:
Moved the section on invalid erasure coding to networking.md.